messages/java/src/main/java/io/cucumber/messages/Messages.java

```diff
@@ -0,0 +1,4018 @@
+package io.cucumber.messages;
+
+public class Messages {
```

Comment:
Consider using `public final` to reduce the API service. And since this is a utility class it can also have a private constructor.

Reply:
Fixed!